### PR TITLE
Update palo-alto-firewall.md - minor tweak to previously submitted PR

### PR DESCRIPTION
**Pull request by mvirga-sumo**
Update GlobalProtect Format

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
Docs Team - though this is approved, please allow @mvirga-sumo to merge when he's ready.

File: docs/cse/ingestion/ingestion-sources-for-cloud-siem/palo-alto-firewall.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ To create a server profile specifying the log destination, do the following:
 
 [GlobalProtect](https://docs.paloaltonetworks.com/pan-os/10-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/globalprotect-log-fields)
 ```
-,$receive_time,$serial,$type,$subtype,,$time_generated,$vsys,$ip,$user,$datasourcename,$eventid,$repeatcnt,$timeout,$beginport,$endport,$datasource,$datasourcetype,$seqno,$actionflags,$dg_hier_level_1,$dg_hier_level_2,$dg_hier_level_3,$dg_hier_level_4,$vsys_name,$device_name,$vsys_id,$factortype,$factorcompletiontime,$factorno,,,$ugflags,$userbysource
+,$serial,$type,$subtype,$time_generated,$vsys,$eventid,$stage,$auth_method,$tunnel_type,$src_user,$srcregion,$machinename,$public_ip,$public_ipv6,$private_ip,$private_ipv6,$hostid,$serialnumber,$client_ver,$client_os,$client_os_ver,$repeatcnt,$reason,$error,$opaque,$status,$location,$login_duration,$connect_method,$error_code,$portal,$seqno,$actionflags
```
Comment (kimsauce) [inline docs/cse/ingestion/ingestion-sources-for-cloud-siem/palo-alto-firewall.md:104]: @mvirga-sumo we'll await your final signoff before merging

Comment (mvirga-sumo) [reply docs/cse/ingestion/ingestion-sources-for-cloud-siem/palo-alto-firewall.md:104]: @kimsauce  I have submitted the final PR that is comprehensive for this doc - that is PR #2875

**Comment by kimsauce** [comment]
closing in favor of https://github.com/SumoLogic/sumologic-documentation/pull/2875

**Pull request closed without merging**